### PR TITLE
flexibleServerVNetIntegrated

### PR DESCRIPTION
**Pull request by fatima99s**

### CODE REVIEW

File: plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'PostgreSQL Flexible Server VNet integrated',
```
Comment (mehakseedat63) [inline plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js:5]:
```suggestion
    title: 'PostgreSQL Flexible Server VNet Integrated',
```

File: plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    category: 'PostgreSQL Server',
+    domain: 'Databases',
+    description: 'Ensure that PostgreSQL flexible servers has VNet integrated.',
```
Comment (mehakseedat63) [inline plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js:8]:
```suggestion
    description: 'Ensures that PostgreSQL flexible servers have VNet integrated.',
```

File: plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.spec.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        "backupRetentionDays": 7,
+        "geoRedundantBackup": "Disabled",
+        "storageAutogrow": "Disabled"
+    },
+    "network":{
+        "delegatedSubnetResourceId" : "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/cloudsploit-dev/providers/Microsoft.Network/virtualNetworks/omer-virtual-network-test/subnets/default",
+        "privateDnsZoneArmResourceId" : "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/cloudsploit-dev/providers/Microsoft.Network/privateDnsZones/testflexibleserver11.private.postgres.database.azure.com",
+        "publicNetworkAccess": "Disabled"
+    }
+
```
Comment (mehakseedat63) [inline plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.spec.js:32]: use random values and not actual ids

**Review by mehakseedat63 — CHANGES_REQUESTED** [review]

File: plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'PostgreSQL Flexible Server VNet Integrated',
+    category: 'PostgreSQL Server',
+    domain: 'Databases',
+    description: 'Ensures that PostgreSQL flexible servers have VNet integrated.',
+    more_info: 'Configuring PostgreSQL flexible server to operate within a Virtual Network (VNet) offers a myriad of benefits for enhanced security and operational control. By integrating with a VNet, you are proactively safeguarding your server against potential security threats and unauthorized access.',
```
Comment (alphadev4) [inline plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js:9]:
```suggestion
    more_info: 'Configuring PostgreSQL flexible server to operate within a Virtual Network (VNet) offers a myriad of benefits for enhanced security and operational control. By integrating with a VNet allows to proactively safeguard your server against potential security threats and unauthorized access.',
```

File: plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    recommended_action: 'Ensures Vnet (private access) is integrated for PostgreSQL flexible server.',
```
Comment (alphadev4) [inline plugins/azure/postgresqlserver/flexibleServerVNetIntegrated.js:10]:
```suggestion
    recommended_action: 'Ensures VNet (private access) is integrated for PostgreSQL flexible server.',
```

**Review by alphadev4 — APPROVED** [review]

**Review by mehakseedat63 — APPROVED** [review]

**Pull request merged**